html-forms/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML FORMS</title>
+  </head>
+  <h1>Coding Form!</h1>
+
+  <form action="">
+    <label for="email"> Email </label>
+    <div>
+      <input type="email" name="email" id="email" required />
+    </div>
+
+    <div>Why is coding cool?</div>
+    <select name="Options">
+      <option value="Select an option">Select an option</option>
+      <option value="Because its fun">It is fun</option>
+      <option value="Beause it's powerful">Because it's powerful</option>
+      <option value="Because it's interesting">Becuase it's interesting</option>
+    </select>
+    <div>Favorite Language (choose one)</div>
+    <input type="radio" name="coding language" value="HTML" />
+    HTML
+    <input type="radio" name="coding language" value="CSS" />
+    CSS
```

Comment:
This text should be in a `label` element.

Reply:
Issue updated with label elements